src/vm/wren_core.wren

```diff
@@ -94,6 +94,7 @@ class Sequence {
     return result
   }
 
+  join { join("") }
```

Comment:
I think referring to the empty `join()` (`join { join() }`) is better.

In addition, maybe we need to improve the name. Maybe `joined` or `toJoin` or `toJoined`?

Reply:
improved the method :).
About the naming, I'm still not sure,